cmd/broker/main.go

```diff
@@ -314,33 +318,33 @@ func main() {
 		kebConfig.NewConfigMapConverter())
 	componentsProvider := runtime.NewComponentsProvider()
 	gardenerClusterConfig, err := gardener.NewGardenerClusterConfig(cfg.Gardener.KubeconfigPath)
-	fatalOnError(err)
+	fatalOnError(err, logs)
```

Comment:
Is this the only way, to pass always logger to this function? No way to use some `global` logger?

Reply:
I did it as a safe guard to use logger that is passed to methods in which `fataOnError` is invoked in. This way we are keeping all the labels that fataOnError should log.